js/snippetPreviewToggler.js

```diff
@@ -0,0 +1,117 @@
+
+var forEach = require( "lodash/forEach" );
+var domManipulation = require( "./helpers/domManipulation.js" );
+
+var previewModes = {
+	desktop: "",
```

Comment:
Maybe we could use `snippet-editor__view--desktop` for this, so it will be more explicit what previewmode is set.